src/wp-includes/functions.php

```diff
@@ -2690,7 +2690,7 @@ function wp_unique_filename( $dir, $filename, $unique_filename_callback = null )
 		 */
 		if ( $is_image ) {
 			/** This filter is documented in wp-includes/class-wp-image-editor.php */
-			$output_formats = apply_filters( 'image_editor_output_format', array(), $_dir . $filename, $mime_type );
+			$output_formats = apply_filters( 'image_editor_output_format', array(), $_dir . $filename, $mime_type, '' );
```

Comment:
We should call out in the dev note that there is no size when core calls `wp_unique_filename` because this is applied only to the upload image (not the sub-sizes).  might be confusing that it is called this way.

Reply:
@adamsilverstein Good point, the additional parameter for this definitely needs to be explained in the dev note. FWIW, it's not only in `wp_unique_filename` that this is empty, it's always empty when no particular image size is being handled. For example even within `WP_Image_Editor` usage, the parameter will be an empty string for the original image, or the `full` image, or any custom operation (e.g. cropping the site icon). Only interactions that relate to a specific registered image size will result in the parameter being not empty.